mantis-runtime/src/main/java/io/mantisrx/runtime/ScalarToKey.java

```diff
@@ -23,7 +23,7 @@
 import java.util.List;
 
 
-public class ScalarToKey<T, K, R> extends StageConfig<T, R> {
+public class ScalarToKey<T, K, R> extends KeyValueStageConfig<T, K, R> {
```

Comment:
nit: Can you document the types here? It would make it easier to read in the future.